src/app.module.ts

```diff
@@ -1,10 +1,28 @@
 import { Module } from '@nestjs/common';
 import { AppController } from './app.controller';
 import { AppService } from './app.service';
+import { UsersModule } from './users/users.module';
+import { PrismaService } from './prisma/prisma.service';
+import { PrismaModule } from './prisma/prisma.module';
+import { LinksController } from './links/links.controller';
+import { LinksModule } from './links/links.module';
+import { LinksService } from './links/links.service';
+import { AuthModule } from './auth/auth.module';
+import { ConfigModule } from '@nestjs/config';
 
 @Module({
-  imports: [],
-  controllers: [AppController],
-  providers: [AppService],
+  imports: [
+    UsersModule,
+    PrismaModule,
+    LinksModule,
+    AuthModule,
+    ConfigModule.forRoot({
```

Comment:
**praise**: Using `ConfigModule.forRoot` with global settings is a good practice for configuration management.